instat/dlgOneWayFrequencies.vb

```diff
@@ -21,8 +21,7 @@ Public Class dlgOneWayFrequencies
     Private bReset As Boolean = True
     Private bResetSubdialog As Boolean = False
     Private clsSjTab As New RFunction
-    Private clsSjPlot, clsPlotGrid, clsSelect As New RFunction
-    Private clsTableBaseOperator, clsGraphBaseOperator As New ROperator
+    Private clsSjPlot, clsPlotGrid As New RFunction

```

Comment:
I think using pipes will help in **consistency** for the three dialogues, one way,two way and finally three way frequencies. Since three way can only be done by using dplyr pipes where  `groub_by ` function is used. Actually this was why Roger was for the idea of pipes as opposed to functions; based on the idea that as you move from one way to three way freq dialogues, it is a matter of slight advancement and build up of the code for both tables and plots. Due to same structure of the code...